user/pages/02.development/05.troubleshooting.notes/01.audio.issues/audio.md

```diff
@@ -0,0 +1,98 @@
+
+# Audio errors and issues
+If your questions and issues aren't addressed here please reach out to us via the support channels.
+
+## Alsa
+Alsa is the sound architecture on most linux systems but particularly on Ubuntu and Raspbian. Alsa comes with a series of tools used to select a usb audio device and configure them for use with your device.
+
+Most of the Alsa configuration is done on the command line and below are some useful links and configuration files to help you setup and debug your system. These are by no means the end-all commands and tools and you will need to look up configuration files for your own distribution. There are many changes based on linux distribution and release so google is still your best friend.
+
+### Useful audio cli commands and apps
+
+ - aplay -l
+    - command lind audio-player lists audio devices
+ - aplay -L
+    -  list all PCMs defined
+ - lsusb
+    - Should show plugged in usb devices. In particular your audio device.
+    - Mine shows as C-media usb
```

Comment:
Reword as `Shows all plugged in USB devices, including your audio device.`

Reword as `For example, some audio devices shows as C-media USB.`